test/InterversionTests/ClusterTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Client.Documents;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Raven.Tests.Core.Utils.Entities;
+using Xunit;
+
+namespace InterversionTests
+{
+    public class ClusterTests : MixedClusterTestBase
+    {
+        [Fact]
+        public async Task ReplicationInMixedCluster_41Leader()
+        {
+            var (leader, peers, local) = await CreateMixedCluster(new[]
+            {
+                "4.1.0-nightly-20180801-0430",
+                "4.1.0-nightly-20180801-0430"
```

Comment:
We'll be better off by using `[Theory]` and `[InlineData("4.1.0-nightly-20180801-0430", "4.1.0-nightly-20180801-0430"]` for that so we'll be able to add another versions in the future easily